imbi/endpoints/project_ssm_configuration.py

```diff
@@ -66,8 +66,8 @@ async def get(self, *args, **kwargs) -> None:
         if len(tokens) == 0:
             raise web.HTTPError(
                 401,
-                reason=
-                'Not authorized to access SSM in this namespace & environment')
+                reason='Not authorized to access SSM in this namespace & '
+                       'environment')
```

Comment:
I got into the habit of explicitly using parens in cases like this since it makes my editor _do the right thing_
```suggestion
                reason=('Not authorized to access SSM in this namespace & '
                        'environment'))
```